runtime/jcl/common/mgmtthread.c

```diff
@@ -1344,6 +1360,14 @@ getThreadInfo(J9VMThread *currentThread, J9VMThread *targetThread, ThreadInfo *i
 
 	/* this may block on vm->managementDataLock */
 	getContentionStats(currentThread, targetThread, info);
+#if JAVA_SPEC_VERSION >= 19
+	/* getStackFramePCs need to run on the correct java stack to produce the right output */
```

Comment:
still unclear. which thread's stack is required? also, end sentences with a full-stop.